### ISSUE TITLE
Add External Property Name Mapping

### ISSUE THREAD

**Issue by TikiBill** [moh-hassan/odata2poco#39]
This is a wonderful program. Thank you.

Microsoft Dataverse/Common Data Service has a feature(?) in that it will automatically name columns in the tables for you based on the name given, and there is no way for the end user to change it. It also ensures uniqueness across the workspace, resulting in some peculiar names. For example, one might add "First Name" to a table, and if it was used before it might get named something like "crf9sa_firstname" as the column name. Not really the name of a property I want to see no matter the capitalization rule applied.

The display name can be changed, so it is not a safe bet to use that as a property name. Rather, it would be great if an external file could be used to map the column name to an arbitrary property name. Bonus points if it supports a regex e.g. triggered when the source name starts with a ^. And double bonus points if there is a way to print out a template file to be used as a starting point.

E.g.:

```json
{
  { "crf9sa_firstname", "FirstName" },
  { "^.*$firstname.*$", "FirstName" }
}
```

An auto generated template file might be like:

```json
{
  { "crf9sa_firstname", "crf9sa_firstname" }
}
```

for easy editing.

I'm happy to have a go at this if it sounds like a feature you would be willing to merge.

Thanks again.

**Comment by moh-hassan** [comment]
Thanks @TikiBill for your suggestion.
It's a great feature and welcome your PR.

**Comment by moh-hassan** [comment]
@TikiBill 
The feature is released in [v4.1.0](https://www.nuget.org/packages/OData2Poco.CommandLine/4.1.0)
Wiki is updated with new entry: [NameMapping](https://github.com/moh-hassan/odata2poco/wiki/NameMapping)
